### ISSUE TITLE
Generated contract for nullable types are wrong

### ISSUE THREAD

**Issue by xwipeoutx** [gkinsman/Typescriptr#3]
Generate types for the class:
```
public class Foo {
  int? Bar { get; set; }
}
```
Expected
```
interface Foo {
  bar: number | null;
}
```
Actual
```
interface Foo {
  bar?: number
}
```

Note that in TypeScript, `bar?: number` is the same as `bar: number | undefined`, so it's not quite the same semantically.

I'm currently using a workaround:
```
using(var fs = File.Create(Path.Combine(rootPath, "types.d.ts")))
using(var tw = new StreamWriter(fs)) {
    tw.Write(result.Types.Replace("?: number;", ": number | null"));
}
```
:boom:

**Comment by gkinsman** [comment]
This is an interesting one - it might need to be configurable, as with JSON.NET, it's possible to set the behaviour of nulls to be ignored, which would therefore lead to an undefined, though that applies to all types. Hmm. 

The default behaviour should be as you say though, will fix.

**Comment by gkinsman** [comment]
Fixed in `1.1.0`
